src/legacy/core_plugins/data/public/search/search_bar/components/saved_query_manager.tsx

```diff
@@ -0,0 +1,360 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import {
+  EuiPopover,
+  EuiPopoverTitle,
+  EuiButtonEmpty,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPagination,
+  EuiText,
+  EuiConfirmModal,
+  EuiOverlayMask,
+  EuiScreenReaderOnly,
+  EuiIconTip,
+} from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+import React, { FunctionComponent, useEffect, useState, Fragment } from 'react';
+import { sortBy } from 'lodash';
+import { SavedQuery } from '../index';
+import { getAllSavedQueries, deleteSavedQuery } from '../lib/saved_query_service';
+import { Query } from '../../../query';
```

Comment:
See my comment about importing types from\in data plugin

Reply:
👍 763f350